### PR TITLE
set up readme to update automatically upon publish of new plugins/ext…

### PR DESCRIPTION
**Pull request by cchang-vassar**
…ensions

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -37,3 +37,7 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Update README list of plugins and extensions
+        run: npm run update-readme.js
+
```
Comment (jodeleeuw) [inline .github/workflows/release.yml:43]: Don't forget to add the commit and push actions here.

File: packages/extension-countdown/package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "directory": "packages/extension-countdown"
   },
   "author": "Shaobin Jiang",
+  "authorGitHub": "https://github.com/Shaobin-Jiang",
```
Comment (jodeleeuw) [inline packages/extension-countdown/package.json:31]: I think we should make this compliant with how npm expects it:

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#people-fields-author-contributors

File: templates/cli/src/index.js [hunk 3]
```diff
@@ -86,12 +86,17 @@ async function runPrompts() {
     required: true,
   });
 
+  const authorGitHub = await input({
```
Comment (jodeleeuw) [inline templates/cli/src/index.js:89]: Do we want to allow for an alternative link to another kind of profile?

File: .github/workflows/release.yml (hunk 1 above)
Comment (bjoluc) [reply .github/workflows/release.yml:43]: Quickly dropping a note here that we can just make the changesets action do the commit work for us:
```diff
    "update-unpkg-links": "gulp updateUnpkgLinks",
+   "update-readme": "node update-readme.js",
    "prepare": "husky install",
    "changeset": "changeset",
-   "changeset:version": "changeset version && npm install && npm run update-unpkg-links",
+   "changeset:version": "changeset version && npm install && npm run update-unpkg-links && npm run update-readme",
```

File: templates/cli/src/index.js (hunk 3 above)
Comment (cchang-vassar) [reply templates/cli/src/index.js:89]: as in, in addition to the github, aka prompt them twice? or to make the prompt more general so they are not asked to put in a github profile link specifically, but any link?

Comment (jodeleeuw) [reply templates/cli/src/index.js:89]: Right, any link in one prompt. Maybe suggest GitHub as an example.

File: package.json [hunk 4]
```diff
@@ -10,12 +10,12 @@
     "test:watch": "npm test -- --watch",
     "build": "npm run build -ws",
     "update-unpkg-links": "gulp updateUnpkgLinks",
+    "update-readme": "node ./update-readme.js",
     "prepare": "husky install",
     "changeset": "changeset",
-    "changeset:version": "changeset version && npm install && npm run update-unpkg-links",
+    "changeset:version": "changeset version && npm install && npm run update-unpkg-link && npm run update-readme",
```
Comment (jodeleeuw) [inline package.json:16]: I think you deleted an s by mistake: `update-unpkg-links`

**Pull request merged**